index.js

```diff
@@ -37,9 +37,18 @@ function installXgboost(artifacts_version, xgboost_version) {
   executeCommand("sudo ldconfig", "fail to ldconfig");
 }
 
-function installLibbpf() {
-  core.info(`libbpf`);
-  executeCommand("sudo apt-get install -y libbpf-dev", "fail to install libbpf related package");
+function installLibbpf(libbpf_version) {
+  core.info(`installing libbpf version ` + libbpf_version + ` from source`);
+  executeCommand("mkdir -p temp-libbpf");
+  executeCommand("cd temp-libbpf");
+  executeCommand("git clone https://github.com/libbpf/libbpf");
+  executeCommand("git checkout "+ libbpf_version);
+  executeCommand("cd libbpf/src");
```

Comment:
would this actually switch directory? wouldn't the directory be back to current once the command finishes executing? IOW, doesn't this just open a shell, cd and then close the shell?